driver/defs.h

```diff
@@ -126,7 +126,7 @@
 #define ESODBC_DSN_SAMPLE_NAME			"Elasticsearch ODBC Sample DSN"
 
 /* SQL plugin's REST endpoint for SQL */
-#define ELASTIC_SQL_PATH				"/_xpack/sql"
+#define ELASTIC_SQL_PATH				"/_sql"
```

Comment:
I think this change should only be made in 7.x, i.e. master.  For this reason it might have been better to do it in a separate PR.